packages/autorest.python/autorest/codegen/models/lro_operation.py

```diff
@@ -128,11 +128,6 @@ def imports(self, async_mode: bool, **kwargs: Any) -> FileImport:
                 "distributed_trace_async",
                 ImportType.AZURECORE,
             )
-        if self.lro_response:
```

Comment:
@tadelesh adding this back because otherwise we don't get a `_deserialize` function import in HealthInsights

Reply:
I see. I think we will add more lro cadl-ranch tests later to cover real cases from services.